.github/workflows/codespell.yml

```diff
@@ -7,8 +7,10 @@ on: [pull_request, push]
 
 jobs:
   codespell:
+
     runs-on: ubuntu-latest
+
     steps:
     - uses: actions/checkout@v3
     - run: pip install codespell
-    - run: codespell NEWS, README.md, README.txt, LICENSE --ignore-words=Misc/codespell/words.ignore
+    - run: codespell NEWS README.md README.txt LICENSE --ignore-words=Misc/codespell/words.ignore
```

Comment:
Maybe try `--skip="*.g,*.ht,*.java,*.py,*.xml" to rapidly get to a reasonable subset of all files.

Reply:
But then we're filtering out most of the places this is could be useful to improve the readability of comments and maybe console messages.

Reply:
It is better to have guardrails in place for those files that should only contain English text than to have no guardrails at all.

Reply:
Yes, but it is easier to list them those to make globs for all the others.